Create ui-c8y-1018-503-54-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md (#1770)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1018-503-54-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md

---
date: ""

> title: Fix inability to switch tabs with multiple widgets using dashboard date context

Reviewer: ```suggestion
title: Switching tabs with multiple widgets using the dashboard date context works smoothly
```

> title: Unable to switch tabs when dashboard has two widgets with dashboard date context (#5682) [GRAFT][release/y2024] (#5721)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fix inability to switch tabs with multiple widgets using dashboard date context
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58128
version: 1018.503.54
---

> Unable to switch tabs when dashboard has two widgets with dashboard date context (#5682) [GRAFT][release/y2024] (#5721)

Author: The Claude AI suggested to use the following body:
```suggestion
Fixed an issue where users were unable to switch between dashboard tabs if the dashboard contained two or more widgets using the dashboard date context filter. Users can now smoothly transition between tabs regardless of the number of dashboard date context widgets present.
```
